apps/server/src/mikro-orm.config.ts

```diff
@@ -0,0 +1,35 @@
+import type { MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs/typings';
+import { ALL_ENTITIES } from '@shared/domain/entity';
+import { FileEntity } from '@modules/files/entity';
+import { FileRecord } from '@modules/files-storage/entity';
+import { DB_PASSWORD, DB_URL, DB_USERNAME } from './config';
+
+export const mikroOrmConfig: MikroOrmModuleSyncOptions = {
+	// TODO repeats server module definitions
+	type: 'mongo',
+	clientUrl: DB_URL,
+	password: DB_PASSWORD,
+	user: DB_USERNAME,
+	entities: [...ALL_ENTITIES, FileEntity, FileRecord],
+	allowGlobalContext: true,
+	/*
+	findOneOrFailHandler: (entityName: string, where: Dictionary | IPrimaryKey) =>
+		new NotFoundException(`The requested ${entityName}: ${JSON.stringify(where)} has not been found.`),
+	*/
+	migrations: {
+		tableName: 'migrations', // name of database table with log of executed transactions
+		path: './dist/apps/server/src/migrations/mikro-orm', // path to the folder with migrations
```

Comment:
```suggestion
		path: './dist/apps/server/migrations/mikro-orm', // path to the folder with migrations
```

Reply:
We need here dynamic path calculation.
const pathToMigrations = resolve(__dirname, '..', 'migrations', 'mikro-orm');

		path: pathToMigrations, // path to the folder with migrations
		pathTs: pathToMigrations, // path to the folder with TS migrations (if used, we should put path to compiled files in `path`)

Reply:
fixed